client-mr/src/main/java/org/apache/hadoop/mapreduce/task/reduce/RssRemoteMergeManagerImpl.java

```diff
@@ -140,6 +141,14 @@ public RssRemoteMergeManagerImpl(String appId, TaskAttemptID reduceId, JobConf j
     this.mergedMapOutputsCounter = mergedMapOutputsCounter;
 
     try {
+      // Use minimized replica, because spilled data can be recomputed by reduce task.
+      // Instead, we use more retries on HDFS client.
+      int replication = remoteConf.getInt(RssMRConfig.RSS_REDUCE_REMOTE_SPILL_REPLICATION,
```

Comment:
There seems to be logic error.

Reply:
remoteConf only contains jobConf, doesn't contain rssConf.

Reply:
Fixed